gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -247,22 +288,6 @@ def get_row_from_variant(variant, split_alternate_allele_info_fields=True):
           alt_record[_get_bigquery_sanitized_field_name(info_key)] = (
               _get_bigquery_sanitized_field(info.data[alt_index]))
     row[ColumnKeyConstants.ALTERNATE_BASES].append(alt_record)
```

Comment:
I agree. However, since we require split_alternate_allele_info_fields here as well, I think it's better to do all of this in the refactoring CL with the class (so that split_alternate_allele_info_fields can be a member variable and we don't need to pass it around).

Reply:
I agree. We should wait on this until rows are refactored to being classes.